packages/discord.js/src/managers/GuildAuditLogManager.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+const { makeURLSearchParams } = require('@discordjs/rest');
+const { Routes } = require('discord-api-types/v10');
+const CachedManager = require('./CachedManager');
+const { DiscordjsTypeError, ErrorCodes } = require('../errors');
+const GuildAuditLogs = require('../structures/GuildAuditLogs');
+const GuildAuditLogsEntry = require('../structures/GuildAuditLogsEntry');
+
+/**
+ * An extension for guild-specific application commands.
+ * @extends {CachedManager}
+ */
+class GuildAuditLogManager extends CachedManager {
```

Comment:
Shouldn't the name be `GuildAuditLogEntryManager` since it holds `GuildAuditLogsEntry`?
Btw `GuildAuditLogsEntry` should be renamed to `GuildAuditLogEntry` in v15.